docs/README.md

```diff
@@ -504,23 +540,179 @@ Steps:
+    ```
+
+5. Create `bookinfo` namespace and label it to mark it for injection.
+
+    ```bash
+    kubectl create namespace bookinfo
+    kubectl label namespace bookinfo istio-injection=enabled
+    ```
+
+6. Deploy bookinfo application.
+
+    ```bash
+    kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/bookinfo/platform/kube/bookinfo.yaml
+    ```
+
+7. Review the `IstioRevisionTag` resource after application deployment.
+
+    ```console
+    $ kubectl get istiorevisiontag
+    NAME      STATUS    IN USE   REVISION          AGE
+    default   Healthy   True     default-v1-23-3   2m46s
+    ```
+    Note: `IN USE` field shows 'True', as the tag is now referenced by both active workloads and the bookinfo namespace.
+
+8. Confirm that the proxy version matches the control plane version.
+
+    ```bash
+    istioctl proxy-status 
+    ```
+    The column `VERSION` should match the control plane version.
+
+9. Update the control plane to a new version.
+
+    ```bash
+    kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.24.1"}}'
+    ```
+
+10. Verify the `Istio`, `IstioRevision` and `IstioRevisionTag` resources. There will be a new revision created with the new version.
+
+    ```console
+    $ kubectl get istio
+    NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+    default   2           2       1        default-v1-24-1   Healthy   v1.24.1   9m23s
+
+    $ kubectl get istiorevision
+    NAME              TYPE    READY   STATUS    IN USE   VERSION   AGE
+    default-v1-23-3   Local   True    Healthy   True     v1.23.3   10m
+    default-v1-24-1   Local   True    Healthy   True    v1.24.1   66s
+
+    $ kubectl get istiorevisiontag
+    NAME      STATUS    IN USE   REVISION          AGE
+    default   Healthy   True     default-v1-24-1   10m44s
+    ```
+    Now, both our IstioRevisions and the IstioRevisionTag are considered in use. The old revision default-v1-23-3 because it is being used by proxies, the new revision default-v1-24-1 because it is referenced by the tag, and lastly the tag because it is referenced by the bookinfo namespace.
+
+11. Confirm there are two control plane pods running, one for each revision.
+
+    ```console
+    $ kubectl get pods -n istio-system
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    istiod-default-v1-23-3-c98fd9675-r7bfw    1/1     Running   0          10m
+    istiod-default-v1-24-1-7495cdc7bf-v8t4g   1/1     Running   0          113s
+    ```
+
+12. Confirm the proxy sidecar version remains the same:
+
+    ```bash
+    istioctl proxy-status 
+    ```
+    The column `VERSION` should still match the old control plane version.
+
+13. Restart all the Deployments in the `bookinfo` namespace.
+
+    ```bash
+    kubectl rollout restart deployment -n bookinfo
+    ```
+
+14. Confirm the new version is used in the sidecars.
```

Comment:
```suggestion
14. Confirm the new version is used in the sidecars. Wait a few seconds, it might take a while until the bookinfo pods are restarted.
```